src/frontend/pages/game.tsx

```diff
@@ -0,0 +1,171 @@
+import Head from 'next/head';
+import styles from '../styles/Game.module.css';
+import Slider from 'react-input-slider';
+import {
+  buildStyles,
+  CircularProgressbarWithChildren,
+} from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
+import React, { useEffect, useState } from 'react';
+import Image from 'next/image';
+
+export default function Home() {
+  const [takeVal, setTakeVal] = useState(10);
+  const [timeLeft, setTimeLeft] = useState(10);
+
+  useEffect(() => {
+    let unsubscribe: number | undefined = undefined;
```

Comment:
oh it took me a minute to figure out that unsubscribe is where we're storing the timeout. Could we just call is timeout? or like timeoutId?